lessons/06-css-development-style/README.md

```diff
@@ -545,5 +560,4 @@ Using CSS! Some big things you should emulate:
 - Nice fonts (see [Google fonts](https://www.google.com/fonts))
 - Try to follow principles from the reading
 
-When you're done with this homework and the next ([Homework 7](https://github.com/olinjs/olinjs/blob/master/lessons/07-apis-debugging/HOMEWORK.md)), fill out the [Twoter homework survey](). The survey is your submission for Twoter with OAuth.
-
+When you're done with this homework, fill out the [Twoter part 1 homework survey]().
```

Comment:
Here's the homework survey: https://docs.google.com/forms/d/e/1FAIpQLSdQBOwppmjSrhaTcAGwRudRiPH9qzH2GtsHrKhiAwYEtMTtqw/viewform